HOSTING.md

```diff
@@ -0,0 +1,26 @@
+# Hosting NodeSchool YVR
+
+Are you interested in hosting NodeSchool Vancouver? We have a minimum set of requirements, and things other people have provided.
+
+## Minimum requirements
+
+* Seating and table space for 20+ people

```

Comment:
Maybe bump that up to 30 people, just to have some breathing room? Or we could put an item in the "nice to have" section with a suggested higher capacity?